addon/-private/system/model/model.js

```diff
@@ -1195,8 +1195,8 @@ if (DEBUG) {
   };
 
   Model.reopen({
-    init() {
-      this._super(...arguments);
+    init(createArgs) {
+      this._super(createArgs);
```

Comment:
`...arguments` should be more robust to refactorings, given that we are reopening

Reply:
I agree w/ @igorT here -- this is a case where you don't want to mix a TS conversion with potential behavioral changes to the code. If it's desirable to move away from `_super(...arguments)`, I'd advise making that change in another PR